src/components/Loader/components/DotsLoader/DotsLoader.style.tsx

```diff
@@ -13,8 +13,15 @@ export const getDotsLayout = (delay: number, animation: string, theme: Theme, le
   animation-delay: ${delay}s;
 `;
 
-export const dotsContainer = () => (theme: Theme): SerializedStyles => css`
+export const dotsWrapper = css`
+  width: 26px;
```

Comment:
can we use rem here as this will possible break in responsive width

Reply:
Since the loader's style depends on pixels I didn't want to change the units to relative just for the fix.

Also I am not sure if we need to include relative units for components that behave like loaders.

Any thoughts on this?

Reply:
hmm i see your point since this is small and will always depond in this but again if we use a `meta` tag to define the viewport and device width this will be very small